script/extract-schema.js

```diff
@@ -62,3 +62,4 @@ convertTable('Signatures', schema.systemSignaturesTable, 'signatures', 'signatur
 // convertTable('QueryRecords', schema.systemFormTableColumns, 'query-records', schema.systemFormViewColumns);
 // convertTable('QueryChildRecords', schema.systemRepeatableTableColumns, 'query-child-records');
 // convertTable('QueryValues', schema.systemValuesTableColumns, 'query-values');
+convertTable('RecordLinks', schema.systemSignaturesTable, 'signatures', 'signatures', schema.organizationViews.signatures);
```

Comment:
I believe this should reference the new stuff you added for record links instead of signatures.

Reply:
TBH though, I'm not even sure where this script is used.

Reply:
I think this script can be removed now. It was used to generate the very first set of the files from the definition, so it was a one time script and not necessary anymore. 💣 it in a separate pr?

Reply:
If it's just deleting this one file, @toddrun could probably 💣 it here.

Reply:
Thanks for catching this! I must have missed a commit. 

Killing the file now . . .